src/components/AddTransaction/index.js

```diff
@@ -0,0 +1,89 @@
+import {useState} from 'react';
+import {useParams} from 'react-router-dom';
+
+import useStore from '../../hooks/useStore';
+import {Form} from '../AddNewAccount/styled';
+
+export default function AddTranscation() {
+	const [transaction, setTransaction] = useState({
+		date: '',
+		change: '',
+		note: '',
+		category: '',
+	});
+
+	const {accountID} = useParams();
+	const addTransaction = useStore(state => state.addTransaction);
+
+	return (
+		<>
+			<Form
+				onSubmit={event => {
+					event.preventDefault();
+					addTransaction(accountID, transaction);
+					setTransaction({date: '', change: '', note: '', category: ''});
+				}}
+			>
+				<label htmlFor="date">Date</label>
+				<input
+					id="date"
+					value={transaction.date}
+					onChange={event => {
+						setTransaction({...transaction, date: event.target.value});
+					}}
+					type="date"
+					aria-label="Enter the date of the transaction"
+					placeholder="e.g 29.06.2022"
+				></input>
+
+				<label htmlFor="changeValue">change value</label>
+				<input
+					id="changeValue"
+					value={transaction.change}
+					onChange={event => {
+						setTransaction({...transaction, change: event.target.value});
+					}}
+					type="text"
+					pattern="([0-9]+)([,\\.]{1}[0-9]+)"
+					aria-label="Enter the change value for that transaction"
+					placeholder="500"
+				></input>
+
+				<label htmlFor="note">Write a describtive note</label>
+				<input
+					id="note"
+					value={transaction.note}
+					onChange={event => {
+						setTransaction({...transaction, note: event.target.value});
+					}}
+					type="text"
+					aria-label="Enter a decribtive note for that transaction"
+					placeholder="e.g. online shopping"
+				></input>
+
+				<label htmlFor="categorieMenu">Select a fitting category</label>
+				<input
```

Comment:
Why don't you use `<select>` for the selection of categories?
[MDN <select>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)

Reply:
We weren´t aware of that :D